src/main/scala/uk/gov/hmrc/SbtArtifactory.scala

```diff
@@ -97,70 +91,71 @@ object SbtArtifactory extends sbt.AutoPlugin{
     repoKey := artifactoryRepoKey(sbtPlugin.value, makePublicallyAvailableOnBintray.value),
     publishMavenStyle := !sbtPlugin.value,
     publishTo := maybeArtifactoryUri.map { uri =>
-      if (sbtPlugin.value)
-        Resolver.url(repoKey.value, url(s"$uri/${repoKey.value}"))(ivyStylePatterns)
-      else
-        "Artifactory Realm" at s"$uri/${repoKey.value}"
+      val pattern = if (sbtPlugin.value)
+                      Resolver.ivyStylePatterns
+                    else
+                      Resolver.mavenStylePatterns
+      Resolver.url(repoKey.value, url(s"$uri/${repoKey.value}"))(pattern)
     },
     credentials ++= {
-      sLog.value.info(
-        s"SbtArtifactoryPlugin - Configuring Artifactory... " +
-          s"Host: ${maybeArtifactoryUri.getOrElse("not set")}. " +
-          s"User: ${maybeArtifactoryUsername.getOrElse("not set")}. " +
-          s"Password defined: ${maybeArtifactoryPassword.isDefined}")
-
+      val credString = // DirectCredentials.toString is not implemented for sbt 0.13
+                       maybeArtifactoryCredentials.map(cred => s"DirectCredentials(${cred.realm}, ${cred.host}, ${cred.userName}, ****)")
+      sLog.value.info(msg(name.value, s"Configuring Artifactory credentials: $credString"))
```

Comment:
it's tricky since they need to be embedded in tasks - don't really want to get into macros here

Reply:
we could defined log as
```
def log(f: Logger => (=> String) => Unit, msg: String) sbt.Def.Initialize[sbt.Task[Unit]] =
    Def.task {
      f(sLog.value)(s"SbtArtifactoryPlugin [${BuildInfo.version}] (${name.value}) - $msg")
    }
```
but the provided msg could not refer to any other variables, since they can only be accessed in a task too...